src/dex/hooks/hederaTokenService/useTokenNFTs.tsx

```diff
@@ -3,17 +3,17 @@ import { DexService, MirrorNodeTokenNFT } from "@dex/services";
 import { HTSQueries } from "./types";
 import { useDexContext } from "@dex/hooks";
 
-type UseTokenQueryKey = [HTSQueries.AccountTokenNFTs, string];
+type UseTokenQueryKey = [HTSQueries.AccountTokenNFTs, string, string];
 
-export function useTokenNFTs(tokenId: string) {
+export function useTokenNFTs(tokenId: string, safeId?: string) {
```

Comment:
we can make this `string | undefined` and `enabled:  !!(tokenId && safeId)`

Reply:
string | undefined is same as optional params. and we can not use it to check enabled, because it will fail in the cases when we are not passing safeId and using user's wallet instead.